lib/iex/lib/iex.ex

```diff
@@ -222,6 +222,16 @@ defmodule IEx do
       ...(1)> #iex:break
       ** (TokenMissingError) iex:1: incomplete expression
 
+  ## Exiting the shell
```

Comment:
When IEx starts we already say how to quit the shell. Plus, the current docs on `h` are focused on `h`'s functionality, so I think this is good where it is right now.